store/src/java-test/com/zimbra/cs/filter/AddHeaderTest.java

```diff
@@ -499,9 +499,16 @@ public void testAddHeaderNameAsMultipleSpace() {
     @Test
     public void testAddHeaderNameStartingWithSpace() {
```

Comment:
It seems this case actually tests not the header **name** with space, but the header **value** with space (\"  Val13\").  How about update the test case name as well?